tests/libs/util/native_helper.cpp

```diff
@@ -20,6 +23,19 @@ _native_module_helper::initialize(
 #else
     ebpf_execution_type_t system_default = ebpf_execution_type_t::EBPF_EXECUTION_ANY;
 #endif
+    // Clean up any previous state.
+    // Detach all bpf links
```

Comment:
```suggestion
    // Detach all bpf links.
```
nit: Still missing the punctuation here.  (You fixed line 26 but line 27 still has same nit.)